File: CursoPOOUber/PHP/Card.php
<?php
require_once('Payment.php');
class Card extends Payment{
    public $number;
    public $cvv;
    public $date;

    public function __construct($id,$number,$cvv,$date){//Constructor
        parent::__construct($id);//Herencia
        this->$number = $number;
        this->$cvv = $cvv;
        this->$date = $date;
        }
    }
?>

File: CursoPOOUber/Java/Card.java
package CursoPOOUber.Java;


public class Card extends Payment {
    String number;
    int cvv;
    String date;
    public Card(int id, String number, int cvv,String date){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
    
}